src/main/java/nextstep/courses/domain/Size.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain;
+
+import javax.swing.*;
+
+public class Size {
+
+    private static final int DEFAULT_WIDTH = 300;
+    private static final int DEFAULT_HEIGHT = 200;
+
+    private final int width;
+    private final int height;
+
+    public Size(int width, int height) {
+        this.width = width;
+        this.height = height;
+        validate();
+    }
+
+    private void validate() {
+        validateSize();
+        validateRatio();
+    }
+
+    private void validateSize() {
+        if(width < DEFAULT_WIDTH && height < DEFAULT_HEIGHT){
+            throw new IllegalArgumentException(
+                    String.format("가로 %d픽셀, 세로 %d픽셀 이상이어야 합니다",
+                            DEFAULT_WIDTH, DEFAULT_HEIGHT)
+            );
+        }
+    }
+
+    private void validateRatio() {
+        double ratio = (double) width / height;
+        double expectedRatio = 3.0 / 2.0;
+
+        if(ratio != expectedRatio){
```

Comment:
전체적으로 코드 포맷팅 확인해주시면 좋을 것 같습니다.
IDE로 IntelliJ IDEA를 사용 중이시라면, 단축키를 활용해보시면 좋을 것 같습니다.
https://www.jetbrains.com/guide/java/tutorials/reading-code/formatting/
```suggestion
        if (ratio != expectedRatio) {
```

Reply:
알려주신 자동 정렬 단축키 적용했습니다!!